modules/services/espanso.nix

```diff
@@ -1,19 +1,22 @@
 { pkgs, config, lib, ... }:
-
 let
-
   inherit (lib)
-    mkOption mkEnableOption mkIf maintainers literalExpression types platforms;
+    mkOption mkEnableOption mkIf maintainers literalExpression types platforms
+    mkRemovedOptionModule versionAtLeast;
 
   inherit (lib.hm.assertions) assertPlatform;
 
   cfg = config.services.espanso;
+  espansoVersion = cfg.package.version;
 
   yaml = pkgs.formats.yaml { };
-
 in {
-  meta.maintainers = with maintainers; [ lucasew ];
-
+  imports = [
+    (mkRemovedOptionModule [ "services" "espanso" "settings" ]
+      "Use services.espanso.configs and services.espanso.matches instead.")
+  ];
+  meta.maintainers =
+    [ lib.hm.maintainers.liyangau maintainers.bobvanderlinden ];
```

Comment:
Why the change of maintainer?

Reply:
I copied the maintainer part from another file.  I fix it with the correct ones...

Reply:
Speaking of maintainer, do we keep adding new maintainer in a list or in this case, override it so people know who writes the current one?

Reply:
Presumably @lucasew still wants to be a maintainer for the module?